code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm

```diff
@@ -170,10 +170,10 @@
 		remove_self(needed)
 
 /datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
-	if(alien == IS_SLIME)
+	/*if(alien == IS_SLIME) //VOREStation Edit Start. Stops slimes from dying from water.
 		M.adjustToxLoss(6 * removed)
-	else
-		..()
+	else*/
+	..() //VOREStation Edit End.
```

Comment:
You have commented out this entire proc's contents... by editing all but 1 line inside of it (and that one line is commented out)
Just put the entire thing inside a block comment using the whitespace above and below.

Reply:
Resolved. Did a /* */ above water and below fuel, since they're both unneeded/borked